frontend/src/app/modules/status/pages/status.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="mx-auto px-4 py-4 sm:px-8 lg:container 
+            grid grid-cols-1 justify-items-stretch md:grid-cols-3 gap-6 " *transloco="let t">
```

Comment:
Cleaning up the css classes:
```suggestion
<div class="mx-auto lg:container px-4 py-4 sm:px-8
            grid grid-cols-1 md:grid-cols-3" *transloco="let t">
```